uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/BaseKeyCloakManager.java

```diff
@@ -288,4 +318,19 @@ protected void handleResponse(ClientResponse response) {
             }
         }
     }
+
+    protected RolesResource getRolesResource(RealmResource realmResource, boolean useClientRoles) {
+        if (useClientRoles) {
+            return realmResource.clients().get(getClientIdByName(realmResource)).roles();
```

Comment:
Is this always going to return a value or could be null?

Reply:
IMO it shouldn't never return a null value since these function chain just helps to generate the right URI for the SSO calls. The return value of this method which is of type RolesResource is the final URI created.